app/page.tsx

```diff
@@ -1,12 +1,13 @@
 import { Card } from '@/components/Card';
-import { Grid, Timeline } from '@/components/Grid';
+import { Grid } from '@/components/Grid';
 import { Heading, Paragraph, Text } from '@/components/Typography';
+import { TimelineSection } from '@/components/PageSections';
 
 export default function Home() {
   return (
     <main className="bg-black-true text-white">
       <div className="cc">
-        <div className="max-w-[120rem] h-fit">
```

Comment:
I removed this `h-fit` class as it doesn't seem to be doing much. LMK if you want me to keep it for a reason.

Reply:
No reason for the `h-fit`! Thank you for removing :D